QuizApp/Application/SceneDelegate.swift

```diff
@@ -16,6 +17,7 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 
         window = UIWindow(windowScene: windowScene)
         appModule = AppModule(container: container)
+        networkService = container.optional()
```

Comment:
This does not have to be optional.

Reply:
It does, runtime crash occurs when using resolve()